### PR TITLE
adding skeletal functions

### PR DESCRIPTION
**Pull request by sonaalll**
1. Added the classifier class that contains a rough skeleton of all functions with description.
2. Added a main.py which will call this class and its functions

### CODE REVIEW

File: climate_scanner/coarse_article_classifier/coarse_classifier.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Imports
+
+class classifier():
+
```
Comment (Eliotfrazier) [inline climate_scanner/coarse_article_classifier/coarse_classifier.py:4]: Let's consider adding an __init__ function. Here we can assign variables to hold the model in memory, and keep some configuration parameters.

**Review by Eliotfrazier — CHANGES_REQUESTED** [review]
Thanks Sonal, see my comment about an innit constructor in the model class.

Could we also consider what the input/output might look like for each function. In particular let's make sure the predict function (or another interface function if you would rather) corresponds to how we have defined the interface. Today/tomorrow I would like to make changes to the coordinator so that I can send the specified inputs to each component, and I'll expect to see some fake data for the outputs come out in the form we have discussed.

File: climate_scanner/coarse_article_classifier/coarse_classifier.py (hunk 1 above)
Comment (sonaalll) [reply climate_scanner/coarse_article_classifier/coarse_classifier.py:7]: Added

**Review by Eliotfrazier — APPROVED** [review]
Looks good 👍

**Pull request merged**